src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -55,16 +55,69 @@
 				</div>
 			</div>
 
-			<div
+			<!-- TODO: Use incoming TextLink component -->
+			<button
 				v-if="totalItemCount > initialItemLimit"
 				class="tw-text-h4"
 				@click="openLightbox"
 			>
 				{{ seeAllLinkText }}
-			</div>
+			</button>
 		</div>
 
-		<!-- TODO: Build out Lightbox Content -->
+		<kv-lightbox
+			:visible="isLightboxVisible"
+			:title="sectionTitle"
+			@lightbox-closed="isLightboxVisible = false"
+		>
+			<template #header>
+				<h2 class="tw-text-h2">
+					{{ sectionTitle }}
+				</h2>
+				<div class="tw-mb-3">
+					<kv-material-icon
+						class="tw-h-2.5 tw-pointer-events-none tw-inline-block tw-align-middle"
+						:icon="mdiLightningBolt"
+					/>
+					<span>{{ poweredByText }}</span>
+				</div>
+			</template>
+
+			<div class="tw-mb-3 tw-grid tw-grid-cols-2 md:tw-grid-cols-3 tw-gap-2">
+				<!-- TODO: Replace with Generic component for showing Either Team or Lender image and information -->
+				<div
+					v-for="(item, index) in items" :key="index"
+					class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3"
+				>
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
+					<kv-loading-placeholder
+						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
+					/>
+					<div>
+						<span v-if="item.name">{{ item.name }}</span><br>
+						<span v-if="item.lenderPage && item.lenderPage.whereabouts">
+							{{ item.lenderPage.whereabouts }}
+						</span>
+					</div>
+				</div>
+			</div>
+
+			<kv-button
+				v-if="items.length < totalItemCount"
+				class="tw-w-full"
+				@click.prevent="loadMore"
```

Comment:
No need to `click.prevent` anymore with KvButton, can just do `click`  :)

Reply:
Cool, will clean up in some follow up